### DISCUSSION TITLE
Can we have a video controls configuration parameter for setting video title in BP?

### DISCUSSION THREAD

**Discussion by selormsedofia** [jhomlala/betterplayer#771] [Q&A]
Can you add include a controls config parameter for setting video title and other descriptions probably at top left corner of the video as part of the video controls rather than creating a custom control for video title only?

Similar to the controls the title gets hidden after a while and shows when the video is tapped?

**Comment by jhomlala** [comment]
The answer is no. There's no space for that in playback controls layout. Anyway you don't need custom controls, you can simply use event listener to check when controls are visible or not.
